test/excontainers/auto_remove_test.exs

```diff
@@ -0,0 +1,24 @@
+defmodule Excontainers.AutoRemoveTest do
+  use ExUnit.Case, async: true
+
+  alias Excontainers.ResourcesReaper
+  import Support.DockerTestUtils
+
+  @sample_container Docker.Container.new("alpine", cmd: ~w(sleep infinity), auto_remove: true)
+  @expected_timeout_seconds 15
+
+  test "when using auto_remove, the container is removed when it's stopped" do
+    {:ok, container_id} = Docker.Containers.run(@sample_container)
+
+    assert container_exists?(container_id)
+
+    assert :ok = Docker.Containers.stop(container_id)
+
+    refute container_exists?(container_id)
+  end
+
+  defp container_exists?(container_id) do
```

Comment:
I'd consider extracting this into `test/support/docker_test_utils.ex` given the same helper function is also defined in `test/excontainers/resources_reaper_test.exs`, WDYT?

Reply:
Sounds good!